src/compiler/checker.ts

```diff
@@ -30690,6 +30682,11 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
         return jsxNamespace && getSymbol(jsxNamespace.exports!, JsxNames.LibraryManagedAttributes, SymbolFlags.Type);
     }
 
+    function getJsxElementTypeSymbol(jsxNamespace: Symbol) {
+        // JSX.ElementType [symbol]

```

Comment:
maybe should be jsdoc 
and I'm not convinced getJsxElementTypeSymbol needs to be its own function -- is it parallel to some other getJSX* functions?

Reply:
This mimics the comment in some of jsx namespace lookup functions, which this function does paralllel.